lib/internal/blob.js

```diff
@@ -127,17 +113,15 @@ class Blob extends JSTransferable {
       return src;
     });
 
-    // This is a MIME media type but we're not actively checking the syntax.
-    // But, to be fair, neither does Chrome.
-    validateString(type, 'options.type');
-
     if (!isUint32(length))
       throw new ERR_BUFFER_TOO_LARGE(0xFFFFFFFF);
 
     super();
     this[kHandle] = createBlob(sources_, length);
     this[kLength] = length;
-    this[kType] = RegExpPrototypeTest(/[^\u{0020}-\u{007E}]/u, type) ?
+
+    type = `${type}`;
+    this[kType] = RegExpPrototypeTest(disallowedTypeCharacters, type) ?
       '' : StringPrototypeToLowerCase(type);
```

Comment:
FYI, It might be wrong to lowercase the hole type, but there is a pending spec change about that